src/installation/guides/zfs.md

```diff
@@ -0,0 +1,271 @@
+# ZFS On Root
+
+This installation guide assumes that an already existing Full Void Linux install
+is available. Not just a Live USB.
+
+If an existing Void install does not exist please consult one of the [other
+installation guides](./index.md) or use a custom iso such as
+[hrmpf](https://github.com/leahneukirchen/hrmpf) then continue along with this
+guide.
+
+## Setup
+
+From the permanent system, install the `zfs` package.
+
+Then ensure the ZFS module is loaded with
+[modprobe(1)](https://man.voidlinux.org/modprobe):
+
+`# modprobe zfs`
+
+Locate the disk to format using [fdisk(8)](https://man.voidlinux.org/fdisk) or
+[lsblk(8)](https://man.voidlinux.org/lsblk):
+
+`# fdisk -l`
+
+## Partitioning
+
+Prepare the drive for installation by creating one of the following partition
+schemes using [cfdisk(8)](https://man.voidlinux.org/cfdisk) or another
+partitioning tool:
+
+`# cfdisk -z /dev/sda`
+
+> **Warning:**
+> 
+> The disk being partitioned will be formatted and any existing data on the disk
+> will be destroyed.
```

Comment:
For warnings/important notices, prefer to use:

```
**Warning**: The disk...
```

So, no quotes and all in the same line.